### PR TITLE
Filter repository by Status on Activation Key page

### PR DESCRIPTION
**Pull request by maximiliankolb**
#### What changes are you introducing?

Document missing filter on AK page.

#### Why are you introducing these changes? (Explanation, links to references, issues, etc.)

follow-up to Katello PR https://github.com/Katello/katello/pull/11114

#### Anything else to add? (Considerations, potential downsides, alternative solutions you have explored, etc.)

Can I consolidate this? What's our preferred way?
````
$ rg "dropdown list" | wc -l
1
$ rg "dropdown menu" | wc -l
12
$ rg "drop-down menu" | wc -l
7
$ rg "drop-down list" | wc -l
5
````

#### Checklists

* [ ] I am okay with my commits getting squashed when you merge this PR.
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.12/Katello 4.14 (Satellite 6.16)
* [x] Foreman 3.11/Katello 4.13

### CODE REVIEW

File: guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@ You can enable or disable repositories on an activation key in the {ProjectWebUI
 . In the {ProjectWebUI}, navigate to *Content* > *Lifecycle* > *Activation Keys*.
 . Select an activation key.
 . Select the *Repository Sets* tab.
-. From the dropdown, you can filter the *Repository type* column to *Custom* or *Red Hat*, if desired.
+. Optional: Clear the *Limit to Environment* checkbox to only views repositories that are available in the lifecycle environment of the activation key.
```
Comment (apinnick) [inline guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc:10]:
```suggestion
. Optional: Clear the *Limit to Environment* checkbox to view repositories that are available in the lifecycle environment of the activation key.
```

**Review by apinnick — CHANGES_REQUESTED** [review]
Please fix typo.

File: guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc (hunk 1 above)
Comment (maximiliankolb) [reply guides/common/modules/proc_enabling-and-disabling-repositories-on-activation-key.adoc]: :see_no_evil: Applied!

**Review by maximiliankolb — COMMENTED** [review]
Fixed typo and rebased to "master".

**Review by sbernhard — APPROVED** [review]
Thanks @maximiliankolb

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked:
   ea2fcfbe7b..4c89e91afb  3.12 -> 3.12
   c50e378436..a49867d911  3.11 -> 3.11

**Pull request merged**